src/webview/js/generatedEs3.js

```diff
@@ -467,17 +467,15 @@ documentBody.addEventListener('click', function (e) {
 });
 
 var handleLongPress = function handleLongPress(target) {
-  var heldTime = !lastTouchEventTimestamp || Date.now() - lastTouchEventTimestamp;
```

Comment:
You should squash this with the previous commit.

Reply:
Done! :)